baseapp-drf-view-action-permissions/testproject/settings.py

```diff
@@ -0,0 +1,55 @@
+"""
+Django settings for testproject project.
+
+Generated by 'django-admin startproject' using Django 3.2.16.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.2/ref/settings/
+"""
+
+from baseapp_core.tests.settings import *  # noqa
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.2/howto/deployment/checklist/
+
+ALLOWED_HOSTS = ["127.0.0.1", "localhost"]
+
+
+# Application definition
+
+INSTALLED_APPS += [
+    "baseapp_drf_view_action_permissions",
+    "testproject.testapp",
+]
+
+MIDDLEWARE = [
+    "django.middleware.security.SecurityMiddleware",
+    "django.contrib.sessions.middleware.SessionMiddleware",
+    "django.middleware.common.CommonMiddleware",
+    "django.middleware.csrf.CsrfViewMiddleware",
+    "django.contrib.auth.middleware.AuthenticationMiddleware",
+    "django.contrib.messages.middleware.MessageMiddleware",
+    "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    "baseapp_drf_view_action_permissions.middleware.RestrictIpMiddleware",
+]
```

Comment:
Reuse as much as possible middlewares from core/tests/settings
```suggestion
MIDDLEWARE += [
    "baseapp_drf_view_action_permissions.middleware.RestrictIpMiddleware",
]
```